### ISSUE TITLE
LFO x EG TRIGENV normalling not normal ?

### ISSUE THREAD

**Issue by jpnielsen** [surge-synthesizer/surge-rack#586]
The TRIG/GATE inputs on LFO+EG are normalled to 5V - "normal" behaviour for a envelope generate is to output 0V.

When live patching, remember to turn down the volume of the modules that are attenuated by the LFO+EG, before removing the TRIGENV cable.

I think it would be preferable to normal to 0V - and/or an option to set the EG output to 0 or 5 V regardless of input.

<img width="1304" alt="image" src="https://user-images.githubusercontent.com/27916597/200292721-f63b1a98-a89b-4a6c-b792-4606f3a3bca7.png">

The TRIG inputs are really GATE inputs - but that's a common mixup.

**Comment by baconpaul** [comment]
yeah i agree the envelope without a trigger connected is strange and counterintuitive.

i want to have a freerun mode without a trigger of course, but in that case i may zero out the envelope.

i'll think about this today or tomorrow. thanks for the detailed report.

**Comment by baconpaul** [comment]
OK I took a look at this. You are right it is counterintuitive albeit correct

I wanted to maintain the constraint that the LFOxEG output was LFO output * EG output.

But in the untriggered freerun mode that EG output is confusing as heck

So I've added an option in the menu labeled 'Set EG to Zero when No Trigger Connected' which is on by default. Will be in an upcoming nightly. This issue will auto close when i merge the change and the nightly follows about 30 minutes later.
